src/smartplaylists/searchterm.cpp

```diff
@@ -23,6 +23,13 @@
 
 namespace smart_playlists {
 
+const char* esc_clause = " ESCAPE \"\\\"";
+
+// Replace % and _ with \% and \_
+static QString Escape(QString term) {
+  return term.replace(QRegExp("([%_])"), "\\\\1");
```

Comment:
Escaping with something other than `\` might make this easier to read.

Reply:
Changed to #. Also realized that we need to escape the escape character since we're not always encoding.